README.md

```diff
@@ -5,7 +5,7 @@ Backend for [github.com/alphahorizonio/consumat.io](https://github.com/alphahori
 [![hydrun CI](https://github.com/alphahorizonio/consumat.io-backend/actions/workflows/hydrun.yaml/badge.svg)](https://github.com/alphahorizonio/consumat.io-backend/actions/workflows/hydrun.yaml)
 [![Docker CI](https://github.com/alphahorizonio/consumat.io-backend/actions/workflows/docker.yaml/badge.svg)](https://github.com/alphahorizonio/consumat.io-backend/actions/workflows/docker.yaml)
 [![Matrix](https://img.shields.io/matrix/consumatio:matrix.org)](https://matrix.to/#/#consumatio:matrix.org?via=matrix.org)
-[![Docker Pulls](https://img.shields.io/docker/pulls/alphahorizonio/consumat.io-backend?label=docker%20pulls)](https://hub.docker.com/r/alphahorizonio/consumat.io-backend)
+[![Docker Pulls](https://img.shields.io/docker/pulls/alphahorizonio/consumat.io-backend?label=docker%20pulls)](https://hub.docker.com/r/alphahorizonio/consumatio-backend)
```

Comment:
Please also change this to `https://img.shields.io/docker/pulls/alphahorizonio/consumatio-backend` :)